stratum/hal/lib/bcm/bcm_chassis_manager.cc

```diff
@@ -469,6 +486,24 @@ ::util::Status BcmChassisManager::SetPortHealthState(uint64 node_id,
   return ::util::OkStatus();
 }
 
+::util::Status BcmChassisManager::SetPortLoopbackState(uint64 node_id,
+                                                       uint32 port_id,
+                                                       LoopbackState state) {
+  if (state == LoopbackState::LOOPBACK_UNKNOWN) {
+    return ::util::OkStatus();
+  }
```

Comment:
This check is not needed, sdk_wrapper checks for `!=0`.

Reply:
This check prevents additional calls to `SetPortOptions` in tests that check for exact numbers.